.github/workflows/actions.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [created]
+jobs:
+  format:
+    name: Check the code format
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Generally, if we don't have a very specific use case, I suggest to use  to use in `steps` the same official dev environment (image) that we distribute to developers/devcontainers as in:
https://github.com/bhack/tensorflow/pull/2

This will really help to minimize the misalignment risk between the prepared environment where people are coding PR on their own hw resourcers/cloud and the CI env.

Reply:
Ops too late.. you have already merged this.

Reply:
P.s. I've fixed the link to the PR.